src/map/atcommand.cpp

```diff
@@ -7373,7 +7373,8 @@ ACMD_FUNC(mobinfo)
 	}
 	for (k = 0; k < count; k++) {
 		unsigned int j,base_exp,job_exp;
-		mob = mob_db(mob_ids[k]);
+		std::shared_ptr<s_mob_db> mob = mob_db.find(mob_ids[k]);
```

Comment:
I think we need a nullptr check after this.
I mean say I do
`@mobinfo 4029 34444 2344`
If I get the code right, mobdb_searchname_array will return me 2, (cause 4029 and 2344) are valid mob id. (or let's pretend they are).
But 34444 is invalid, which the  mob_db.find(34444), will return me nullptr.
Since they're is no check we will crash on the next line.